### DISCUSSION TITLE
Pinout numbering

### DISCUSSION THREAD

**Discussion by cmat21** [ra3xdh/qucs_s#394] [Q&A]
Hi, how can i see pinout number?
![oamp_pinout](https://github.com/ra3xdh/qucs_s/assets/70080894/51449c53-67c9-4277-ac2f-9228fc82ba7b)

**Comment by ra3xdh** [comment]
You cannot see the pinout for library devices. This mode is not supported. All library opamp follows the the following rule: upper pin +VCC; left lower pin -VDD

**Reply by cmat21** [reply]
Now it's clear, following the rule. Maybe you could consider adding simple rules to the text box.
![oamp_pinout2](https://github.com/ra3xdh/qucs_s/assets/70080894/c8185ab2-a668-4fae-b8bd-3b23fbe0f4f9)
